src/neo4j_graphrag/schema.py

```diff
@@ -304,7 +312,7 @@ def get_structured_schema(
         for data in query_database(
             driver=driver,
             query=REL_PROPERTIES_QUERY,
-            params={"EXCLUDED_LABELS": EXCLUDED_RELS},
+            params={"EXCLUDED_LABELS": EXCLUDED_RELS, "SAMPLE": sample},
```

Comment:
Some unit tests probably need to be updated (I think about `test_get_structured_schema_happy_path` for instance)